src/test/resources/fixtures/hmrc-frontend/additional-fixtures/new-tab-link-no-language/output.txt

```diff
@@ -1 +1 @@
-<a class="govuk-link " target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/working-tax-credit/eligibility">find out if you qualify for tax credits</a>
+<a class="govuk-link " target="_blank" rel="noopener noreferrer" href="https://www.gov.uk/working-tax-credit/eligibility">find out if you qualify for tax credits </a>
```

Comment:
when I was running my test of generate fixtures for scalaj this popped up as a small change for the fixtures so I have included it